src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -45,29 +62,29 @@ public function __construct()
 
     public function getDefinition(): FixerDefinitionInterface
     {
+        $codeSample = <<<'PHP'
+            <?php
+            /**
+             * @covers \VendorName\Foo
+             * @internal
+             */
+            final class FooTest extends TestCase {
+                /**
+                 * @param int $expected
+                 * @param int $actual
+                 * @dataProvider giveMeSomeData
+                 * @requires PHP 8.0
+                 */
+                public function testSomething($expected, $actual) {}
+            }
+
+            PHP;
+
         return new FixerDefinition(
             'PHPUnit attributes must be used over their respective PHPDoc-based annotations.',
             [
-                new VersionSpecificCodeSample(
-                    <<<'PHP'
-                        <?php
-                        /**
-                         * @covers \VendorName\Foo
-                         * @internal
-                         */
-                        final class FooTest extends TestCase {
-                            /**
-                             * @param int $expected
-                             * @param int $actual
-                             * @dataProvider giveMeSomeData
-                             * @requires PHP 8.0
-                             */
-                            public function testSomething($expected, $actual) {}
-                        }
-
-                        PHP,
-                    new VersionSpecification(8_00_00),
-                ),
+                new VersionSpecificCodeSample($codeSample, new VersionSpecification(8_00_00)),
+                new VersionSpecificCodeSample($codeSample, new VersionSpecification(8_00_00), ['keep_annotations' => true]),
```

Comment:
```suggestion
                new VersionSpecificCodeSample($codeSample, new VersionSpecification(7_04_00), ['keep_annotations' => true]),
```

Shouldn't we use 7.4 here, as this is real world example for this config option?

Reply:
No, we should not add PHP attributes in PHP below 8.0, because what is practical is not the most important In this fixer, but adding attributes is.

Also, I'm pretty sure it is not possible to have a passing pipeline with this change.

Reply:
Why would it affect the pipeline? I mean, you can add attributes for code supporting 7.4-8.* and on 7.4 these will be considered as comments, while the PHPUnit's version also can differ and depending on which is installed it can utilise attributes or annotations. This is the whole point of this option, right? For PHP 8.0+ it rather does not make sense to keep annotations.

Reply:
Because (...).

I still stand with "I'm pretty sure it is not possible to have a passing pipeline with this change" - please, raise a PR against this feature's branch with thechange and I will be more than happy to merge it (obviously with passing CI).

Reply:
hint: if you know this won't fly - you know why it won't fly. instead of writing "just check it yourself" you could give the reason, avoiding multiple ppl to check the thing you are aware of 🙇🏻

Reply:
Sure, I'm the bad guy here. Suggesting change that will break CI is fine, not wanting it is me being asshole...

Reply:
@kubawerlos it's not about being an asshole or not, it's just pretty weird to suggest creating a PR over PR to figure out something you already figured out... We can save each other's time by just providing explicit info instead of expecting others to spend their time on something not actually needed ☺️. I had some doubt, wanted to discuss it, but let's close this thread if there's no room for it 🤷‍♂️.